homeassistant/components/ecobee/humidifier.py

```diff
@@ -43,6 +43,40 @@ def __init__(self, data, thermostat_index):
 
         self.update_without_throttle = False
 
+    @property
+    def name(self):
+        """Return the name of the humidifier."""
+        return self._name
+
+    @property
+    def unique_id(self):
+        """Return a unique identifier for the humidifier."""
+        return f"{self.data.ecobee.get_thermostat(self.thermostat_index)['identifier']}-humidifier"
+
+    @property
+    def device_info(self):
+        """Return device information for the ecobee humidifier."""
+        thermostat = self.data.ecobee.get_thermostat(self.thermostat_index)
+        try:
+            model = f"{ECOBEE_MODEL_TO_NAME[thermostat['modelNumber']]} Thermostat"
+        except KeyError:
+            _LOGGER.error(
```

Comment:
We don't allow logging in entity properties. If the log is needed it needs to move to a method eg `async_update`.

Reply:
Logging removed from this location. I believe that logging in other areas will be sufficient to identify issues.